Sources/Plasma/PubUtilLib/plPhysX/plSimulationMgr.cpp

```diff
@@ -292,11 +293,20 @@ plSimulationMgr::plSimulationMgr()
 bool plSimulationMgr::InitSimulation()
 {
     fSDK = NxCreatePhysicsSDK(NX_PHYSICS_SDK_VERSION, NULL, &gErrorStream);
-    if (!fSDK)
+    if (!fSDK) {
+        fLog->AddLine("Phailed to init PhysX SDK");
         return false; // client will handle this and ask user to install
+    }
 
     fLog = plStatusLogMgr::GetInstance().CreateStatusLog(40, "Simulation.log", plStatusLog::kFilledBackground | plStatusLog::kAlignToTop);
-    fLog->AddLine("Initialized simulation mgr");
+    fLog->AddLine("Initialized PhysX SDK");
+
+    if (!NxInitCooking(nullptr, &gErrorStream)) {
```

Comment:
You have to redistribute NxCooking.dll with your app, which is what I've been doing with Korman. It works as long as the user has the legacy PhysX package installed :smile: I have not seen any prohibition on redistributing this file. I could be mistaken, however.